java/src/jmri/jmrix/roco/z21/Z21TrafficController.java

```diff
@@ -318,6 +318,10 @@ public void handleOneIncomingReply() throws java.io.IOException {
         Runnable r = new RcvNotifier(msg, mLastSender, this);
         try {
             javax.swing.SwingUtilities.invokeAndWait(r);
+        } catch (java.lang.InterruptedException ie) {
+            if(threadStopRequest) return;
+            log.error("Unexpected exception in invokeAndWait:" + ie);
```

Comment:
If you call printStackTrace(), you are potentially not logging it. If style is more important than ensuring it is logged, then so be it.